server/middleware/admin.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable consistent-return */
+const admin = (req, res, next) => {
```

Comment:
rename this to admin

Reply:
I have renamed it to isAdmin. I don't know if that's what is expected of me here sir?

Reply:
yes exactly